UI/Selenium/Steps/QuickLinkSteps.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
```

Comment:
This is required for what?

I'd not remove it now it is there but it does nothing at this time.

Reply:
Removed, we are not using it.